### DISCUSSION TITLE
A certain package is crashing Atom - to whom i need to open a issue

### DISCUSSION THREAD

**Discussion by HadasKharilker** [atom/atom#22802] [General]
Upon uninstall and install of a package the system is crashing, is it an ATOM issue or the package issue?

**Comment by icecream17** [comment]
which package

**Comment by maleksware** [comment]
This depends on the situation. Please describe your problem more precisely, as developers don't know what problem that you are facing is.

If you know what package is crashing Atom, uninstall it manually or from Atom itself. Create issue at this package's page on GitHub.

If you don't know exactly why is your Atom crashing, use safe mode: `atom --safe` in command line. Then try to turn packages on and off until you find the one that makes you feel angry.
